bls/Cargo.toml

```diff
@@ -40,6 +41,7 @@ solana-keypair = { workspace = true }
 [features]
 bytemuck = ["dep:bytemuck"]
 frozen-abi = ["dep:solana-frozen-abi", "dep:solana-frozen-abi-macro"]
+fs = ["dep:serde_json"]
```

Comment:
nit: I looked around briefly, and the only example I could see to gate `fs` usage was the ed25519_dalek crate, and it used the `std` feature for that.

I'm not sure if you plan on making this crate `no-std` compatible, but it might actually be possible in this crate, in which case, maybe calling the feature `std` does make more sense.

Reply:
Yeah good idea! I renamed it to `std` and took a stab at making the crate `no-std` compatible. Please let me know if my changes make sense.